modules/@apostrophecms/asset/lib/webpack/apos/webpack.config.js

```diff
@@ -86,3 +87,16 @@ module.exports = ({
 
   return merge(config, ...tasks);
 };
+
+function getVueAlias() {
+  if (!process.env.APOS_DEV) {
+    return '@vue/runtime-dom';
+  }
+
+  const vueProjectLevelPath = path.join(process.cwd(), 'node_modules/@vue/runtime-dom');
+  const vueProjectLevelInstalled = fs.existsSync(vueProjectLevelPath);
+
+  return vueProjectLevelInstalled
+    ? vueProjectLevelPath
+    : path.resolve(__dirname, '../../../../../../node_modules/@vue/runtime-dom');
```

Comment:
```suggestion
    : path.resolve(./node_modules/@vue/runtime-dom');
```

this should work, and be safe if somehow we move `webpack.config.js`

Reply:
It won't work because for example if my project is called apos-wp, the path that you provide will give:
`/home/user/Documents/workspace/apo/a3-wp/node_modules/@vue/runtime-dom`
While mine will give
`/home/user/Documents/workspace/apo/apostrophe/node_modules/@vue/runtime-dom`

I tried with a path to the `apostrophe` module:
`path.resolve('./apostrophe/node_modules/@vue/runtime-dom')`

but unfortunately it doesn't work with links...